pkg/controller/apiserver/handlers/handler_test.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/vmware-tanzu/antrea/pkg/controller/networkpolicy"
+	v1 "k8s.io/api/admission/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/vmware-tanzu/antrea/pkg/controller/apiserver/handlers/endpoint"
+	queriermock "github.com/vmware-tanzu/antrea/pkg/controller/networkpolicy/testing"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
```

Comment:
we usually group  imports as follows (you may want to fix it before opening the upstream PR, otherwise someone is likely to comment about it):
 * go standard imports
 * whiteline
 * all third-party imports
 * whiteline
 * antrea imports